tests/pytests/unit/utils/test_listdiffer.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+
+from salt.utils import dictdiffer
+from salt.utils.listdiffer import list_diff
```

Comment:
```suggestion
import salt.utils.listdiffer
```

Reply:
Leaving it as is, prefer just getting list_diff, doing ```import salt.utils.listdiffer``` by itself doesn't work, and don't want to make additional changes to working code.

Prefer just testing the one function rather than dragging the kitchen sink in